src/HPWH.cc

```diff
@@ -2853,17 +2905,21 @@ double HPWH::getStandbyLosses(UNITS units /*=UNITS_KWH*/) const
     }
 }
 
-double HPWH::getTankHeatContent_kJ() const
+double HPWH::getTankTemp_C() const
```

Comment:
Maybe call "getTankAverageTemp_C" to make it clearer?

Reply:
There was already a related tankAvg_C function. Instead, I organized the various getTankTemp_C functions with similar names and purposes.

Reply:
I still feel that the method names should indicate that they are getting the **average** tank temperature, though I suppose this would also break the interface.

Reply:
OK, I'll make the change.